### ISSUE TITLE
Simple CNN accuracy too low

### ISSUE THREAD

**Issue by kashefy** [gangchill/nip-convnet#2]
Possible factors:

- [bias initialization](https://github.com/gangchill/nip-convnet/blob/master/models/cnn/simple_cnn.py#L88)  mu=0.1 seems too high. Recommend going lower (e.g. 0.001)
- [Dropout not used during training](https://github.com/gangchill/nip-convnet/blob/master/models/cnn/simple_cnn.py#L149). Currently set to 1.0, which seems to be intended for testing but it's also being used during training instead of something like 0.5 during training

Non critical - but learning is bit slow. A combination of this may help:

- increase optimizer learning rate. But don't go too high, otherwise it'll oscillate. (e.g. 0.01)
- increase batch size. More stable gradients (e.g. 128), computes faster too.

**Comment by maxkohlbrenner** [comment]
Updated the simple CNN implementation.
In our previous version, we applied an activation function on the logits. After removing this function, the network achieves an accuracy of 0.982599973679 after 3000 training iterations (batchsize 50 and dropout rate of 0.5). The choice of architecture and training parameters was based on the deep mnist for experts tf tutorial (https://www.tensorflow.org/get_started/mnist/pros) to validate the current implementation. Training is currently running, I will close the issue if the described accuracy of 99,2% is attained.
